pftp/client_handler.go

```diff
@@ -172,8 +179,6 @@ func (c *clientHandler) getResponseFromOrigin() error {
 
 	// close origin connection when close goroutine
 	defer func() {
-		c.isDone = true
```

Comment:
why `is.Done` flag delete in this PR?

Reply:
Because, isDone absolutely do nothing after PR #34.

https://github.com/pyama86/pftp/pull/34/files#diff-3ef3c58b1a6dfa2bbd1218d3dc18e684974e6815e0ba738ea20c8bd28de22f22L546-L551

It was garbage of previous commit.